plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotConnectorSmokeTest.java

```diff
@@ -969,6 +989,146 @@ public void testIntegerType()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testJsonExtractIsNull()
+    {
+        assertThat(query("SELECT lucky_number FROM " + JSON_TABLE + " WHERE JSON_EXTRACT(json, '$.selector') IS NULL"))
+                .isFullyPushedDown();
+
+        assertThat(query("SELECT lucky_number FROM " + JSON_TABLE + " WHERE JSON_EXTRACT_SCALAR(json, '$.selector') IS NULL"))
+                .isFullyPushedDown();
+    }
+
+    @Test
+    public void testJsonContains()
+    {
+        assertThat(query("SELECT lucky_number FROM " + JSON_TABLE + " WHERE CONTAINS(ARRAY['vendor1'], JSON_EXTRACT_SCALAR(json, '$.vendor'))"))
+                .matches("VALUES (INTEGER '5')")
```

Comment:
This query can be `VALUES 5`. The following `isFullyPushedDown` verifies results with&without pushdown, though.

Reply:
fixed